index.html

```diff
@@ -50,9 +55,9 @@ <h3 class="cover-links">LET'S CONNECT</h3>
     </ul>
   </section>
 </section>
-<section class="container">
+<section id="project-section">
     <!-- First Project -->
-  <section id="card-page" class="work-page-left">
+  <!-- <section id="card-page" class="work-page-left">
     <div class="card1-image">
       <img src="images/works/tonic/Cover1.svg" alt="Poster for multi-post story app" style="width: 100%;" />
     </div>
```

Comment:
- [ ] Kindly avoid committing old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history :100: . So please remove all the commented pieces of code from your project :+1: